tex/texlive-basic/Portfile

```diff
@@ -13,15 +13,15 @@ license             Copyleft Permissive
 description         TeX Live: Essential programs and files
 long_description    These files are regarded as basic for any TeX system, covering plain TeX macros, Computer Modern fonts, and configuration for common drivers\; no LaTeX.
 
-checksums           texlive-basic-44435-run.tar.xz \
-                    rmd160  d559ff32322d346375078a953124e3500d30d8d3 \
-                    sha256  47ec78ffc398ab98728a224218457c41f44482dcbd99d00d2a87a16f650c8a17 \
-                    texlive-basic-44435-doc.tar.xz \
-                    rmd160  28a376169c8999e332eca0d42e083e2c5d36bb48 \
-                    sha256  497fd18272015da1002e2833fffa989e9bb4d494cc7675e6e2f8c927fb27f816 \
-                    texlive-basic-44435-src.tar.xz \
-                    rmd160  dcd0ae5fe4c861ab2e6a4fd536ab272e36d745c2 \
-                    sha256  bb8e6c4dfb03497865c542201372243d323fc811b452e248aac94d0e8da4ad4d
+checksums           texlive-basic-47434-run.tar.xz \
+                    rmd160  db9269a12e1efcf9606b92cedb1eb675bc3c2017 \
+                    sha256  24e7db599421c175f31526370e3c98886d422eeb305d774b3ab221783d95e6e4 \
+                    texlive-basic-47434-doc.tar.xz \
+                    rmd160  432242993facb78e2286f7eef10e70855d69f382 \
+                    sha256  c860614acadbe0409d9a7000de6d7604d30cb098a09112b070feb23370583cab \
+                    texlive-basic-47434-src.tar.xz \
+                    rmd160  908d10d2a0e750eff9e02b917be85c9d8d055259 \
+                    sha256  6ec3d03346618eca9e50ce078108ae65380eedb5f7270064804823cb0da50791
```

Comment:
Not sure if I want to nitpick here. Usually the number would be replaced by `${version}` string, but it's a lot more annoying to update that when so many files are present because it would need to be manually fixed. So exception to our general rule is probably ok.